src/anders_mathiesen_ex/ex_04/walker.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Anders Mathiesen'
+__email__ = 'andermat@nmbu.no'
+
+
+import random
+
+
+def walk_multiple_times(times):
+    walk_list = []
+    for _ in range(times):
+        walk.go_home()
+        walk_list.append(walk.get_steps())
+        walk.reset()
```

Comment:
what is `walk`?

Reply:
This is not good function design. Here your function modifies something that is outside its scope. This way of thinking can work with methods modifying something within a class, but should never be applied to stand-alone functions. This should be changed to `def walk_multiple_times(walker, times):`

Come by on Friday for in-depth explanation.